src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerHandler.cs

```diff
@@ -0,0 +1,93 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Management.Common;
+using Microsoft.SqlServer.Management.Sdk.Sfc;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.ObjectManagement.Contracts;
+using Microsoft.SqlTools.ServiceLayer.ObjectManagement.ObjectTypes.Server;
+using Microsoft.SqlTools.ServiceLayer.ServerConfigurations;
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// Server object type handler
+    /// </summary>
+    public class ServerHandler : ObjectTypeHandler<ServerInfo, ServerViewContext>
+    {
+        private const string serverNotExistsError = "Server was not created for data container";
```

Comment:
Could you create a localized string for this if you're planning on keeping the error? You can add new localized strings to this file: https://github.com/microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/Localization/sr.strings
Then you can regenerate the associated SR files by running build.cmd from the project root.